### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by myusuf3**
- added dev dependency grunt
- added .travis.yml
- added build status to image. 
- this tests 3 versions of node (probably not needed but doesn't hurt)

Once this is merged in activate this repository on your travis-ci.org/profile page.

### CODE REVIEW

**Comment by toin0u** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![travis](https://travis-ci.org/fabien-d/delorean.png?branch=master)
```
Comment (fabien-d) [inline README.md]: just need to update delorean with alertify

**Comment by myusuf3** [comment]
fixed!

File: package.json [hunk 2]
```diff
@@ -7,6 +7,12 @@
 		"name"  : "Fabien Doiron",
 		"email" : "fabien.doiron@gmail.com"
 	},
+	"scripts": {
+		"test": "grunt"
```
Comment (fabien-d) [inline package.json]: one more thing... probably makes more sense to do `grunt lint qunit`. Don't really need to run concat and min for the travis build.

**Pull request merged**